rust_src/remacs-sys/lib.rs

```diff
@@ -124,6 +124,17 @@ pub enum PseudovecType {
     PVEC_FONT, /* Should be last because it's used for range checking.  */
 }
 
+#[repr(isize)]
```

Comment:
Thanks a ton for the review! I'll be glad to make this change. I used repr(isize) to match ```PseudovecType```. Maybe I should also change that to ```repr(c)```?